packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -411,6 +411,13 @@ export default function VisualEditor( { styles } ) {
 										: `${ blockListLayoutClass } wp-block-post-content` // Ensure root level blocks receive default/flow blockGap styling rules.
 								}
 								layout={ blockListLayout }
+								dropZoneElement={
+									// When iframed, pass in the html element of the iframe to
+									// ensure the drop zone extends to the edges of the iframe.
+									isToBeIframed
+										? ref.current?.parentNode
+										: ref.current
+								}
```

Comment:
Interesting, I'd have expected this to be `contentRef` but I guess it works because `contentRef` includes `ref`? I'm assuming that what matters is a shared ref between BlockCanvas and dropZoneElement, right?

Reply:
Yep! Yeah, interestingly enough it's `contentRef` in the site editor because that's the name of the direct ref, whereas in the post editor `contentRef` is the merged ref. So, `ref` contains the `.current` reference to the node. From a little digging (I _think_ this is how it works) in the post editor `contentRef` is what gets passed to `BlockCanvas` as it contains a callback function that will update the list of refs to be merged, but we need to use `ref.current` to grab a reference to the element itself. If we attempt to access `contentRef` directly (or pass it anywhere else), we just get the callback function rather than the element, as that's what's [returned by `useMergeRefs`](https://github.com/WordPress/gutenberg/blob/6ffdde0670c687dee4b0569853ff76ef656961f1/packages/compose/src/hooks/use-merge-refs/index.js#L114).

Reply:
Oooh I see, thanks!